lib/router/src/cache.rs

```diff
@@ -144,21 +151,8 @@ where
     type Error = ();
```

Comment:
TIOLI: I think the Item type can now be Never.

Reply:
`spawn` takes a `Future<Item= (), Error = ()>`